parser.go

```diff
@@ -98,6 +98,9 @@ type Attribute struct {
 	// This will be populated during unmarshalling.
 	DefaultValue *Value `parser:"" json:"default_value,omitempty"`
 
+	// Enum will be parse from tag as a list
+	Enum *Value `parser:"" json:"enum,omitempty"`
```

Comment:
I think this should be a `[]*Value` or maybe even a `[]string`. There's no point having the extra indirection as it can never be anything but a `[]`.

Reply:
I'll get this into a `[]*Value`